gems/sorbet-runtime/lib/types/private/methods/decl_builder.rb

```diff
@@ -5,18 +5,18 @@ module T::Private::Methods
   Declaration = Struct.new(:mod, :params, :returns, :bind, :mode, :checked, :finalized, :on_failure, :override_allow_incompatible, :type_parameters, :raw)
 
   class DeclBuilder
-    attr_reader :decl
+    attr_reader :__decl__
```

Comment:
We would want an error saying that type parameter names can't start with a `_` to avoid someone declaring instance variables.